crates/db_views/src/comment_view.rs

```diff
@@ -325,7 +325,7 @@ impl<'a> CommentQuery<'a> {
 
     if self.local_user.is_some() {
       // Filter out the rows with missing languages
-      query = query.filter(local_user_language::id.is_not_null());
```

Comment:
Went through all these null and not_null checks, and made sure they all worked on one of the columns that was joined to.